### ISSUE TITLE
Restrict the file size to 100 mb for consultation files

### ISSUE THREAD

**Issue by rithviknishad** [coronasafe/care#1141]
**Is your feature request related to a problem? Please describe.**
There's no size limit for consultation files.

**Describe the solution you'd like**
Limit file size to 100 MB for consultation files.

**Comment by siddnikh** [comment]
@rithviknishad do we also want to restrict the file extensions for this?

**Comment by rithviknishad** [comment]
@siddnikh No. Only file size restriction is required as of now, since we do not have clear limits on what all file extensions are allowed.
Also, since doctors are uploading these files, it's very unlikely that they'll misuse this flaw.

**Comment by Ashesh3** [comment]
Fixed in https://github.com/coronasafe/care_fe/pull/4560
